src/app/component/quiz/quiz.component.html

```diff
@@ -1,53 +1,37 @@
-<app-header></app-header>
-<div
-  id="carouselExampleControlsNoTouching"
-  class="carousel slide"
-  data-bs-touch="false"
-  data-bs-interval="false"
->
-  <div class="carousel-inner">
-    <div
-      id="carouselExampleControls"
-      data-bs-interval="false"
-      data-wrap="false"
-      class="carousel slide"
-      data-ride="carousel"
-    >
-      <form [formGroup]="myForm">
-        <button type="submit" (click)="submit()">Submit</button>
-
-        <div formArrayName="form">
-          <ngb-carousel
-            #carousel
-            *ngIf="name"
-            [showNavigationArrows]="false"
-            [showNavigationIndicators]="false"
-            [interval]="0"
-            [keyboard]="false"
+<div class="carousel slide" data-bs-touch="false" data-bs-interval="false">
```

Comment:
what is use for data-bs-touch and data-bs-interval

Reply:
also please remove carousal slide too as now we are using ng boostrtap crousla

Reply:
Yes  i removed it